units/crons.go

```diff
@@ -109,17 +109,24 @@ func PopulateEventAlertProcessing(env evergreen.Environment) amboy.QueueOperatio
 			return nil
 		}
 
-		err = dispatchUnprocessedNotifications(ctx, queue, flags)
-		grip.Error(message.WrapError(err, message.Fields{
-			"message": "unable to dispatch unprocessed notifications",
-		}))
-		notifySettings := env.Settings().Notify
-		if err = notifySettings.Get(env); err != nil {
-			return errors.Wrap(err, "unable to get notify settings")
+		return dispatchUnprocessedNotifications(ctx, queue, flags)
```

Comment:
I trust the process but I'm a little unclear on why we dispatch notifications both here and in the event notifier job.

Reply:
I didn't write this, but I'm pretty sure this is just a performance boost to reduce the waiting time between creating the notification and actually sending the notification.